components/styled/icon/src/atoms/emotion-icons/chevron-double-icon.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, HTMLAttributes, MouseEvent } from 'react'
+import { IconDirection } from '@ltht-react/styles'
+import {
+  ChevronDoubleRight,
+  ChevronDoubleLeft,
+  ChevronDoubleUp,
+  ChevronDoubleDown,
+} from '@emotion-icons/heroicons-solid'
+
+const calculateIconType = (direction: IconDirection): IconTypes => {
+  switch (direction) {
+    default:
+    case 'up':
+      return ChevronDoubleUp
+    case 'down':
+      return ChevronDoubleDown
+    case 'left':
+      return ChevronDoubleLeft
+    case 'right':
+      return ChevronDoubleRight
+  }
+}
+
+const ChevronDoubleIcon: FC<Props> = ({ size, direction, clickHandler, className, ...rest }) => {
+  const handleClick = (e: MouseEvent<SVGSVGElement>): void => {
+    e.preventDefault()
+    e.stopPropagation()
+    clickHandler && clickHandler()
+  }
+
+  const Icon = calculateIconType(direction)
+  return (
+    <>
+      <Icon
+        className={`${className ?? ''} icon__chevron`.trimStart()}
+        size={size}
+        onClick={clickHandler && handleClick}
+        {...rest}
+      />
+    </>
+  )
+}
+
+type IconTypes =
+  | typeof ChevronDoubleUp
+  | typeof ChevronDoubleDown
+  | typeof ChevronDoubleLeft
+  | typeof ChevronDoubleRight
+
+interface Props extends HTMLAttributes<SVGElement> {
+  direction: IconDirection
+  size: string | number
+  clickHandler?(): void
```

Comment:
For accessibility reasons we shouldn't be making tags clickable unless they satisfy the contract of a button/anchor etc. (because screen readers won't announce them correctly).
 
If you take a look at IconButton.tsx I've wrapped the icon in a `<button>` tag and applied some css to remove the styling. 
You could implement the contract yourself but it's generally easier to just use a <button> tag, as explained in the following guide:
https://benmyers.dev/blog/clickable-divs/